crates/store/re_grpc_server/src/lib.rs

```diff
@@ -44,6 +44,10 @@ pub const DEFAULT_MEMORY_LIMIT: MemoryLimit = MemoryLimit::UNLIMITED;
 const MAX_DECODING_MESSAGE_SIZE: usize = u32::MAX as usize;
 const MAX_ENCODING_MESSAGE_SIZE: usize = MAX_DECODING_MESSAGE_SIZE;
 
+// roughly 1 MiB of messages
+const MESSAGE_QUEUE_CAPACITY: usize =
+    (1024 * 1024 / std::mem::size_of::<Msg>()).next_power_of_two();
```

Comment:
This effectively multiplies the previous value with 8… that seems low.

Reply:
I can't repro your issue no matter what I try, so I'm not sure what the "proper" fix is. The previous limit was 1024 messages, and the logs you posted show around 50 messages being dropped, so I figured x8 was enough.

Reply:
My screenshot show several hundred messages dropped. I rather be safe than sorry here - we can easily afford 16MiB for these pipes.

Reply:
I'm not going to close the issue, because we should investigate _why_ that is happening instead of only increasing the queue size and hoping that is enough. This should be a temporary bandaid fix